README.md

```diff
@@ -125,6 +125,12 @@ Retries are configurable using the following flags
       --retry-min-time duration   Minimum wait interval (default 200ms)
 ```
 
+
+Append group if iam role or iam user already exists
```

Comment:
Would be good if we can also add an example for `--update-username=false` use case

Reply:
on it